sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java

```diff
@@ -473,7 +481,6 @@ private PlannerResult planExplanation(
    *
    * @param rel Instance of the root {@link DruidRel} which is formed by running the planner transformations on it
    * @return A string representing an array of native queries that correspond to the given SQL query, in JSON format
-   * @throws JsonProcessingException
```

Comment:
This was only to remove a non-informative Javadoc comment since the code itself tells us that the exception is thrown. Do we like having these "boilerplate" entries? We don't have them for parameters, it seems.